mlir/tools/mlir-tblgen/OpDefinitionsGen.cpp

```diff
@@ -2641,8 +2641,7 @@ void OpEmitter::genSeparateArgParamBuilder() {
 
       // Avoid emitting "resultTypes.size() >= 0u" which is always true.
       if (!hasVariadicResult || numNonVariadicResults != 0)
-        body << "  "
-             << "assert(resultTypes.size() "
+        body << "  " << "assert(resultTypes.size() "
```

Comment:
Probably can remove the second << here and merge the strings, not sure its doing much here (if it were handling indentation other story)